api/DOMImplementation.json

```diff
@@ -23,10 +23,10 @@
             "version_added": "6"
           },
           "opera": {
-            "version_added": true
+            "version_added": "15"
```

Comment:
Can you double check this? Most of document.implementation should have been in Presto too.

Reply:
Didn't have a custom test for this, will introduce one in v1.1.8.  I'll adjust the data.

Reply:
Thanks!